src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,211 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Override
+    public Optional<Car> create(Car car) {
+        String query = "INSERT INTO `cars` (model, manufacturers_id) VALUE (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement preparedStatement = connection.prepareStatement(query,
+                         Statement.RETURN_GENERATED_KEYS)) {
+            preparedStatement.setString(1,car.getModel());
+            preparedStatement.setLong(2,car.getManufacturer().getId());
+            preparedStatement.executeUpdate();
+            ResultSet generatedKeys = preparedStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                car.setId(generatedKeys.getObject(1, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t insert car " + car + " to DB", e);
+        }
+        insertDrivers(car);
+        return Optional.ofNullable(car);
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        Car car = null;
+        String query = "SELECT c.id car_id, c.model model, m.id model_id, "
+                + "m.country country, m.name name FROM cars c LEFT JOIN "
+                + "manufacturers m ON m.id = c.manufacturers_id WHERE c.id = ? "
+                + "AND c.is_deleted = FALSE;";
```

Comment:
FYI you don't need to create aliases for cases like `c.model model` because in the manufacturers table you don't have a column with the same name, so `c.model` will be enough